src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -389,6 +391,11 @@ def file_issue(testcase,
                                       security_severity):
       issue.labels.add(result)
 
+  # Add additional body text from metadata.
+  additional_fields = json.loads(testcase.get_metadata('additional_fields', ''))
```

Comment:
move this functionality into data_handler.get_issue_description instead. then, add a test similar to the ones here: 

https://github.com/google/clusterfuzz/blob/20a4de8c85595566e4f3ebdfbd49bde7fb090bfd/src/python/tests/core/datastore/data_handler_test.py#L250